docs/modules/deployment-launcher/launch-deployments.md

```diff
@@ -78,22 +97,22 @@ Where N is in range [1, number of simulated player deployments]
 | --- | --- | --- |
 | Snapshot Path | ❌ | Path to the [snapshot](https://docs.improbable.io/reference/latest/shared/glossary#snapshot) you wish to start the deployment with, relative from the root of your SpatialOS project. |
 | Launch Config | ✔️ | Path to the launch configuration you wish to start the deployment with, relative from the root of your SpatialOS project. |
-| Region | ✔️ | The geographical location in which your deployment launches. |
+| Region | ✔️ | The geographical region code that the deployment is running in. |
 | Tags | ❌ | Metadata that can be added to a deployment. Some tags have built-in functionality. For example, the `dev_login` tag is used to connect into deployment through the [development authentication flow](https://docs.improbable.io/reference/latest/shared/auth/development-authentication). |
 | Flag Prefix | ✔️ | The prefix to include on worker flags that the Deployment Launcher adds to simulated player deployments. This should follow [worker flag naming conventions](https://docs.improbable.io/reference/latest/shared/worker-configuration/worker-flags#naming-conventions). |
-| Worker Type | ✔️ | The worker type of the simulated player coordinator, responsible for managing simulated players in the child deployment and connecting them to the parent deployment. |
+| Worker Type | ✔️ | The worker type of the [simulated player coordinator]({{urlRoot}}/reference/glossary#simulated-player-coordinator), responsible for managing [simulated players]({{urlRoot}}/reference/glossary#simulated-player) in the child deployment and connecting them to the parent deployment. |
 
 <%(/Expandable)%>
 
 ### Launch deployments
 
-At the bottom of the deployment configurations panel, there is a **Launch deployment** button. You can select which which deployment configuration you wish to launch and then select **Launch deployment**.
+At the bottom of the deployment configurations panel, there is a **Launch deployment** button. You can select which deployment configuration you wish to launch and then select **Launch deployment**.
```

Comment:
`there is a **Launch deployment** button.` -> `there is the **Launch deployment** button.`